README.md

```diff
@@ -8,7 +8,7 @@ To install `clang+llvm-9` to use only inside `gorshochek` following commands sho
 git clone https://github.com/JetBrains-Research/gorshochek.git
 cd gorshochek
 
-# Install clang+llvm-9
+# Install clang+llvm-9, cpplint
```

Comment:
Update usage instructions to this:

## Usage
1. Clone or download repo from GitHub
```(bash)
git clone https://github.com/JetBrains-Research/gorshochek.git
cd gorshochek
```
2. Install clang with the desired version to use only inside `gorshochek`
```(bash)
./scripts/install_clang.sh
```
3. Build the project from source
```(bash)

cmake -DLLVM_DIR=build/clang+llvm-9/lib/cmake/llvm \
      -DClang_DIR=build/clang+llvm-9/lib/cmake/clang \
      -DCMAKE_CXX_COMPILER=/usr/bin/c++ ..
cmake --build .
```
4. Run the tool with the needed configuration on C++ files
```(bash)
# TODO
```